### DISCUSSION TITLE
go back to previous design for  show details page.

### DISCUSSION THREAD

**Discussion by blackholeearth** [trakt/showly#803] [Polls]
**Describe the bug**
1) Links out of screen. İ use it the most.🤦😱🚨

İ want it to stay at its place.
 And not to move left n right 

 İ prefer to get rid of trailer.    🧹🧹

Previous design was more practical. 🔨🔨 

2) the "people" title over cast photos really unnecessary. Takes up extra space.

İ used to be able to see more info , upon entering show.  without scrolling down..

**Expected behavior**

Tldr----
 lets go back to previous design,
 İn show details .page.
